FinSearchUnified/Resources/views/frontend/listing/actions/action-filter-panel.tpl

```diff
@@ -0,0 +1,13 @@
+{extends file="parent:frontend/listing/actions/action-filter-panel.tpl"}
+
+{block name="frontend_listing_actions_filter_form_page"}
+    {if $finPushAttribs}
+        {foreach $finPushAttribs as $finFilterName => $finFilterValues}
+            {foreach $finFilterValues as $finFilterValue => $finWeight}
+                <input type="hidden" name="pushAttrib[{$finFilterName}][{$finFilterValue}]" value="{$finWeight}"/>
```

Comment:
This may allow XSS in some form or another. Please sanitize all three variables before setting them here.